registration-processor/stage-groups/pom.xml

```diff
@@ -18,4 +18,57 @@
 	<modules>
 		<module>stage-group-1</module>
     </modules>
+    <dependencies>
+    	<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
Do we need this? Since stage pom will already contain the dependencies

Reply:
Adding common dependencies to the parent of the stage group, including the regproc-core, rest and status impl

Reply:
Anyways those are also specificied in the stage pom file rt, so anyway it will pull it, do we need it here also ?